cocoviewer.py

```diff
@@ -196,11 +195,26 @@ def draw_masks(draw, objects, obj_categories, ignore, alpha):
                     if m_:
                         draw.polygon(m_, outline=fill, fill=fill)
             # TODO: Fix problem with RLE
-            # elif isinstance(m, dict):
-            #     draw.polygon(m['counts'][1:-2], outline=c[-1], fill=fill)
+            elif isinstance(m, dict):
```

Comment:
```suggestion
            elif isinstance(m, dict) and objects[i]["iscrowd"]:
```
According to [format-data](https://cocodataset.org/#format-data):
> The segmentation format depends on whether the instance represents a single object (iscrowd=0 in which case polygons are used) or a collection of objects (iscrowd=1 in which case RLE is used).

Let's check that here